src/charm.py

```diff
@@ -173,17 +173,30 @@ def __init__(self, *args):
             refresh_event=[self.on.start],
             jobs=self._generate_metrics_jobs(self.is_tls_enabled),
         )
-        self.loki_push = LogProxyConsumer(
-            self,
-            log_files=POSTGRES_LOG_FILES,
-            relation_name="logging",
-            container_name="postgresql",
+        self.loki_push = (
+            LogForwarder(
+                self,
+                relation_name="logging",
+            )
+            if self._is_pebble_log_forwarding_supported
+            else LogProxyConsumer(
+                self,
+                logs_scheme={"postgresql": {"log-files": POSTGRES_LOG_FILES}},
+                relation_name="logging",
+            )
```

Comment:
Note that, while LogProxyConsumer is streaming everything from
```
POSTGRES_LOG_FILES = [
    "/var/log/pgbackrest/*",
    "/var/log/postgresql/patroni.log",
    "/var/log/postgresql/postgresql*.log",
]
```
using LogForwarder streams the stdout of all pebble services.

The two may differ. If critical, then could add a pebble service analogue to `tail -f` the files.

Reply:
@marceloneppel @shayancanonical @dragomirp, wdyt?

Reply:
It seems like there's a way config pg to log to stdout but I'm not clear on the syntax.
https://www.postgresql.org/docs/current/runtime-config-logging.html#GUC-LOGGING-COLLECTOR

Reply:
I don't think a tailing service or stdout will work for Pgbackrest out of the box. IIRC it creates separate log files for various commands. I don't see [an option](https://pgbackrest.org/configuration.html#section-log) to configure the logfile names. Maybe we can use `--log-level-console` and run the backup commands as a oneshot pebble service?

Reply:
We probably can also use something like `--log-level-console` along with `container.exec([pgbackrest-command])wait_output()`.